<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>326</width>
    <height>305</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/assets/icons/icon.png</normaloff>:/assets/icons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main section */

QWidget {
    background-color: #FFFFFF;
    color: #334155;
}
/* Button Styling */
QPushButton {
	max-height:15px;
    padding: 8px;
    border: 1px solid #E2E8F0;
    border-radius: 8px;
    background-color: #FFFFFF;
    color: #475569;
}

QPushButton:hover {
    background-color: #F0F9FF;
    border-color: #0EA5E9;
    color: #0EA5E9;
}

QPushButton:pressed {
    background-color: #E0F2FE;
    color: #0369A1;
}

/* Input Fields */
QLineEdit{
	max-height:15px;
    padding: 8px 12px;
    border: 1px solid #E2E8F0;
    border-radius: 8px;
    background-color: #FFFFFF;
    color: #334155;
}

QLineEdit:focus{
    border-color: #0EA5E9;
    background-color: #F0F9FF;
}

Line{
	background-color: rgba(40, 180, 244, 15);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QWidget" name="widget_user" native="true">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <bold>false</bold>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_user">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>16</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_user_id">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>3</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="label_username">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:transparent;</string>
          </property>
          <property name="text">
           <string>İstifadəçi adı*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_username">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>33</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <bold>false</bold>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_password">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:transparent;</string>
          </property>
          <property name="text">
           <string>Şifrə*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_password">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>33</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Normal</enum>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <bold>false</bold>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QPushButton" name="button_add">
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>33</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>11</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color:#006EFF;
}
</string>
          </property>
          <property name="text">
           <string>Təsdiqlə</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_cancel">
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>33</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>11</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color:rgb(255, 53, 53);
}</string>
          </property>
          <property name="text">
           <string>Ləğv et</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_delete">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>33</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>11</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/assets/icons/delete.png</normaloff>:/assets/icons/delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
